test/navigation.vader

```diff
@@ -462,3 +465,27 @@ Execute (edit routes):
 
 Expect (generic template):
   <?php
+
+Before (blade file with __() function):
+  tabedit test/fixtures/laravel-8/resources/views/viewfile.blade.php
+
+Do (Move to (a)uth.failed, jump to translation file):
+  i{{ __('auth.failed') }}\<Esc>0fagf
+
+Execute:
+  AssertEqual expand('%'), 'resources/lang/auth.php'
+
+After (clean up buffer):
+  bdelete
+
+Before (blade file with trans() function):
```

Comment:
My understanding is that `Before` and `After` blocks are executed again for every test in the test file, so there should only be one of each per file. You can do additional setup for a single test in an `Execute` block instead.